Dockerfile

```diff
@@ -2,6 +2,4 @@ FROM ubuntu
 
 RUN apt-get update && apt-get install -y s3cmd
 
-VOLUME ["/.s3cmd"]
```

Comment:
Why remove the volume?

Reply:
I removed this initially so I could play with mounting different volumes thinking that was part of the permission denied issue... but I don't think it is.